test/Microsoft.Dnx.Compilation.CSharp.Tests/RoslynCompilerTest.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Versioning;
+using Microsoft.CodeAnalysis;
+using Microsoft.Dnx.Compilation.Caching;
+using Microsoft.Dnx.Runtime;
+#if DNX451
```

Comment:
We might want to add new tests which are not restricted to dnx451. The only reason this is #ifdef'd is that it uses Moq, which doesn't work with CoreCLR.